harness/tests/experiment/fixtures/pytorch_xor_model.py

```diff
@@ -51,22 +51,22 @@ def __init__(self, context):
             nn.Linear(context.get_hparam("hidden_size"), 1),
             nn.Sigmoid(),
         )
-        reset_parameters(self.main_net)
+        pytorch.reset_parameters(self.main_net)
```

Comment:
Question: A very bad consequence of importing functions using only one path and using module would be a situation like this. It looks like we are importing from Pytorch. Maybe it's a place we should use `det_torch` or other options. Or maybe we should just import functions directly without using a module name. @rb-determined-ai Thoughts on it?

Reply:
I'm ok with this, but I would also be ok with:
```
import determined as det
import determined.pytorch
det.pytorch.reset_parameters()
```

The reason I'm ok with what we have now is that it is consistent with the google python coding standards, and it's awfully nice to stick to a published standard and not fight over these things.  If I was going to break the standard though, I would just not import the submodule at all.  Using the full path module path works fine and isn't any longer.